runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MonitoringInfoMetricName.java

```diff
@@ -48,6 +48,8 @@ private MonitoringInfoMetricName(String urn, Map<String, String> labels) {
     // and ensure all necessary labels are set for the specific URN.
     this.urn = urn;
     for (Entry<String, String> entry : labels.entrySet()) {
+      checkArgument(entry.getKey() != null, "MonitoringInfoMetricName keys must be non-null");
```

Comment:
would it be possible to improve static checking of this by removing the suppression of the nullness warnings above?

Reply:
Done.